styles/file-icons.less

```diff
@@ -29,7 +29,7 @@
 .icon-book[data-name="Readme.md"]:before,
 .icon-book[data-name="readme.md"]:before { .medium-blue; }
 
-.file-icons-tab-pane-icon @{pane-tab-selector}, .icon-file-text {
+.file-icons-tab-pane-icon @{pane-tab-selector}, .icon-file-text, .icon-file-binary {
```

Comment:
For some reason `.woff` files have the `.icon-file-binary` instead of `.icon-file-text`